source/gui/addonGui.py

```diff
@@ -378,7 +383,8 @@ def onClose(self,evt):
 		for addon in self.curAddons:
 			if (addon.isPendingInstall or addon.isPendingRemove
 				or addon.isDisabled and addon.isPendingEnable
-				or addon.isRunning and addon.isPendingDisable):
+				or addon.isRunning and addon.isPendingDisable

```

Comment:
I think you could leave out the appArgs line and just change this line:
```suggestion
				or not addon.isDisabled and addon.isPendingDisable
```